docs/src/pages/utils/transitions/SimpleGrow.js

```diff
@@ -39,27 +39,26 @@ class SimpleGrow extends React.Component {
     const { classes } = this.props;
     const { checked } = this.state;
 
+    const polygon = (
```

Comment:
Perhaps not the best choice of variable name...

Reply:
child?

Reply:
child?